UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts

```diff
@@ -31,6 +31,7 @@ export class SingleRendererComponent implements OnInit, OnDestroy, ImageRenderer
   readerModeClass$!: Observable<string>;
   darkenss$: Observable<string> = of('brightness(100%)');
   currentImage!: HTMLImageElement;
+  emulateBookClass$: Observable<string> = of('');
```

Comment:
You can remove of('') too and use ! as you know it will be setup in constructor

Reply:
Resolved with https://github.com/Kareadita/Kavita/pull/1892/commits/3adac186e2c7915565094bdd19320380e8a2aabe